src/main/java/uk/gov/hmcts/juror/api/moj/repository/ActivePoolsRepositoryImpl.java

```diff
@@ -0,0 +1,164 @@
+            .then(1)
+            .otherwise(0).sum();
+
+        JPAQuery<Tuple> query = queryFactory.select(POOL_REQUEST, confirmedFromBureau)
+            .from(POOL_REQUEST)
+            .leftJoin(JUROR_POOL).on(POOL_REQUEST.eq(JUROR_POOL.pool))
+            .where(POOL_REQUEST.owner.eq(SecurityUtil.BUREAU_OWNER))
+            .where(POOL_REQUEST.newRequest.eq('N'))
+            .where(POOL_REQUEST.numberRequested.ne(0))
+            .where(POOL_REQUEST.poolType.description.in(PoolRequestUtils.POOL_TYPES_DESC_LIST))
+            .groupBy(POOL_REQUEST);
+
+        if (StringUtils.isNotBlank(filterQuery.getLocCode())) {
+            query.where(POOL_REQUEST.courtLocation.locCode.eq(filterQuery.getLocCode()));
+        }
+        if (SecurityUtil.isCourt()) {
+            query.where(POOL_REQUEST.courtLocation.locCode.in(SecurityUtil.getCourts()));
+        }
+
+        return PaginationUtil.toPaginatedList(
+            query,
+            filterQuery,
+            PoolRequestedFilterQuery.SortField.POOL_NUMBER,
+            SortMethod.ASC,
+            data -> {
+                PoolRequest poolRequest = Objects.requireNonNull(data.get(POOL_REQUEST));
+                return PoolRequestActiveDataDto.builder()
+                    .poolNumber(poolRequest.getPoolNumber())
+                    .requestedFromBureau(poolRequest.getNumberRequested())
+                    .confirmedFromBureau(data.get(confirmedFromBureau))
+                    .courtName(poolRequest.getCourtLocation().getName())
+                    .poolType(poolRequest.getPoolType().getDescription())
+                    .attendanceDate(poolRequest.getReturnDate())
+                    .build();
+            },
+            ValidationConstants.MAX_ITEMS
+        );
+    }
+
+
+    private PaginatedList<PoolRequestActiveDataDto> getActiveCourtTabRequests(ActivePoolFilterQuery filterQuery) {
+        JPAQueryFactory queryFactory = new JPAQueryFactory(entityManager);
+
+        JPAQuery<Tuple> activePoolsQuery = queryFactory.select(
+                POOL_REQUEST.poolNumber,
+                POOL_REQUEST.totalNoRequired,
+                JUROR_POOL.isActive.count(),
+                POOL_REQUEST.courtLocation.name,
+                POOL_REQUEST.poolType.description,
+                POOL_REQUEST.returnDate
+            )
+            .from(POOL_REQUEST)
+            .leftJoin(JUROR_POOL)
+            .on(POOL_REQUEST.eq(JUROR_POOL.pool).and(POOL_REQUEST.owner.eq(JUROR_POOL.owner)))
+            .where(POOL_REQUEST.owner.ne(SecurityUtil.BUREAU_OWNER))
+            .where(POOL_REQUEST.newRequest.eq('N'))
+            .where(POOL_REQUEST.poolType.description.in(PoolRequestUtils.POOL_TYPES_DESC_LIST))
+            .where(POOL_REQUEST.nilPool.eq(false).and(POOL_REQUEST.returnDate.after(LocalDate.now()))
+                .or(JUROR_POOL.isActive.isTrue()
+                    .and(JUROR_POOL.status.status.in(Arrays.asList(3, 4))
```

Comment:
Could use enumerations rather than literals for status here

Reply:
Agreed will make it more readable change made / pushed